src/entity/entity.cpp

```diff
@@ -171,6 +199,47 @@ void EntitySystem::update() {
     }
 }
 
+void EntitySystem::send_state(ClientHandle *handle) {
+    Package pkg;
+    pkg.header.type = PackageType::EVENT;
+    pkg.EVENT.event.type = EventType::LIGHT_UPDATE;
+    LightUpdate event;
+    if (is_valid(GAMESTATE()->lights[0])) {
+        Light *light = fetch<Light>(GAMESTATE()->lights[0]);
+        event.entity_id = light->entity_id;
+        light->position.to(event.position);
+        light->color.to(event.color);
+        pkg.EVENT.event.LIGHT_UPDATE = event;
+        handle->send(&pkg);
+    }
+    if (is_valid(GAMESTATE()->lights[1])) {
+        Light *light = fetch<Light>(GAMESTATE()->lights[1]);
+        event.entity_id = light->entity_id;
+        light->position.to(event.position);
+        light->color.to(event.color);
```

Comment:
You removed to but didn't remove the usages....

Reply:
Oh dear